src/main/java/net/snowflake/client/jdbc/SnowflakeSQLLoggedException.java

```diff
@@ -243,6 +243,12 @@ public SnowflakeSQLLoggedException(SFBaseSession session, String SQLState, Strin
     sendTelemetryData(null, SQLState, -1, session, this);
   }
 
+  public SnowflakeSQLLoggedException(
```

Comment:
after adding new constructors - can we mark some existing constructors as deprecated when are not used?